Documentation/kubernetes.md

```diff
@@ -112,11 +117,13 @@ At this point, you can use the `rook` tool along with some [simple steps to crea
 ### Teardown
 To clean up all the artifacts created by the demo, run the following:
 ```
-kubectl delete deployment rook-operator wordpress wordpress-mysql
+kubectl delete -f wordpress.yaml
+kubectl delete -f mysql.yaml
+kubectl delete deployment rook-operator
 kubectl delete thirdpartyresources cluster.rook.io
-kubectl delete namespace rook
 kubectl delete storageclass rook-block
 kubectl delete secret rook-rbd-user
+kubectl delete namespace rook
```

Comment:
you would have to ssh into the host machine to clean it up, right? for simplification i think i makes sense not to use hostPath in the main walkthrough